src/BloomTests/XmlHtmlConverterTests.cs

```diff
@@ -58,5 +58,20 @@ public void GetXmlDomFromHtml_HasBrTags_TagsNotDoubled()
 			}
 			Assert.AreEqual(1, found);
 		}
+
+		/// <summary>
+		/// We don't particularly want it to convert newlines to spaces, as this test demonstrates that it does.
+		/// However, that is harmless and unlikely to occur (we're not counting on newlines for formatting)
+		/// so I haven't tried to prevent it.
+		/// I do want to test that an existing newline is not simply removed, leaving no white space.
+		/// </summary>
+		[Test]
+		public void GetXmlDomFromHtml_HasBandItags_NoExtraNewlinesBefore()
```

Comment:
Not worth re-committing, but if you do for some other reason, this name was hard to figure out, because of lack of capitalization on "and" and "tags"